### PR TITLE
add custom config options and change base image

### PR DESCRIPTION
**Pull request by andreaseger**
TODO:
- [x]  validate that the image still builds
- [x]  validate that the custom views injection works as expected
- [x]  validate the custom kubernetes container cap setting

### CODE REVIEW

File: 2/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/rhscl/s2i-core-rhel7
+FROM registry.access.redhat.com/rhel7
```
Comment (andreaseger) [inline 2/Dockerfile]: we probably don't even need the `s2i-core-rhel7` base image and plain `rhel7` works

File: 2/contrib/openshift/base-plugins.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
-openshift-pipeline:1.0.54
-openshift-login:1.0.8
-openshift-client:1.0.12
+openshift-pipeline:latest
```
Comment (andreaseger) [inline 2/contrib/openshift/base-plugins.txt]: we actually want all our plugin to be as uptodate as possible

File: 2/contrib/openshift/configuration/config.xml.tpl [hunk 3]
```diff
@@ -44,13 +44,14 @@
   <views>
     <hudson.model.AllView>
       <owner class="hudson" reference="../../.."/>
-      <name>all</name>
+      <name>All</name>
```
Comment (andreaseger) [inline 2/contrib/openshift/configuration/config.xml.tpl]: just undoing that change to have it the same as upstream again

Comment (nilsding) [reply 2/contrib/openshift/configuration/config.xml.tpl]: jenkins will change that back to lowercase on startup anyway, should probably be changed in upstream

Comment (andreaseger) [reply 2/contrib/openshift/configuration/config.xml.tpl]: but this way if we diff against upstream we don't see this change and have to wonder why this changed

Comment (nilsding) [reply 2/contrib/openshift/configuration/config.xml.tpl]: true that.  Maybe™ we'll create an upstream PR for this at one point.

**Pull request merged**